.make.defaults

```diff
@@ -573,6 +576,18 @@ MINIO_ADMIN_PWD= localminiosecretkey
 	    mv tt.toml pyproject.toml;					\
 	fi
 
+# Updates the Python supported versions field of the pyproject.toml file to the given versions
+# Expects  REQUIRED_PYTHON_VERSIONS
+.PHONY: .defaults.__update-toml-python-versions
+.defaults.__update-toml-python-versions:
+	@# Help: Set the version= field of pyproject.toml
+	if [ -e pyproject.toml ]; then					\
+	    cat pyproject.toml | sed -e 				\
+	    's/^requires-python[ ]*=.*"/requires-python = "'${REQUIRED_PYTHON_VERSIONS}'"/' \
```

Comment:
can you make the white space optional?

Reply:
what do you mean, I think they are.
`[ ]*`

Reply:
ah i'm losing it. sorry, was looking at the substitution value.